### PR TITLE
Copy Git. Rework Linux CLI to Unix Shell

### PR DESCRIPTION
**Pull request by lempiy**
Linux CLI reworked to Unix Shell to fit backend developer requirements

### CODE REVIEW

File: tasks/unix-shell.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+➡️ Go forward to [Git for Team Collaboration](git-collaboration.md)
+
+⤴️ Back to [Contents](../contents.md)
+
+[icon-chat]: https://img.shields.io/badge/chat-on%20telegram-blue.svg
+[icon-mit]: https://img.shields.io/badge/license-MIT-blue.svg
+[icon-awesome]: https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg
+[license]: https://github.com/Kottans/web/blob/master/LICENSE.md
+[awesome]: https://github.com/sindresorhus/awesome#front-end-development
+[chat]: https://t.me/joinchat/CX8EF1JmLm9IM6J6oy2U7Q
```
Comment (suchov) [inline tasks/unix-shell.md]: Are you sure we need the link to the FE chat here?

File: tasks/git-intro.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+➡️ Go forward to [Linux, Command Line, HTTP Tools](linux-cli-http.md)
+
+⤴️ Back to [Contents](../contents.md)
+
+[icon-chat]: https://img.shields.io/badge/chat-on%20telegram-blue.svg
+[icon-mit]: https://img.shields.io/badge/license-MIT-blue.svg
+[icon-awesome]: https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg
+[license]: https://github.com/Kottans/web/blob/master/LICENSE.md
+[awesome]: https://github.com/sindresorhus/awesome
+[chat]: https://t.me/joinchat/CX8EF1JmLm9IM6J6oy2U7Q
```
Comment (suchov) [inline tasks/git-intro.md]: Are you sure we need the link to the FE chat here too? :)

File: contents.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+1.  Product-centric mindset
+
+[icon-chat]: https://img.shields.io/badge/chat-on%20telegram-blue.svg
+[icon-mit]: https://img.shields.io/badge/license-MIT-blue.svg
+[icon-ideas]: https://img.shields.io/badge/google--doc-ideas-ff69b4.svg
+[icon-awesome]: https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg
+[license]: https://github.com/Kottans/web/blob/master/LICENSE.md
+[awesome]: https://github.com/sindresorhus/awesome
+[ideas]: https://docs.google.com/spreadsheets/d/1bZJhYjK3VHOS2HmQb2Fs4aHfEBt8mp1F09j9nEEDaqE/edit#gid=818017811
+[chat]: https://t.me/joinchat/CX8EF1JmLm9IM6J6oy2U7Q
```
Comment (suchov) [inline contents.md]: ;)

**Review by suchov — APPROVED** [review]

File: tasks/unix-shell.md (hunk 1 above)
Comment (lempiy) [reply tasks/unix-shell.md]: @suchov its magically gone :)

File: tasks/git-intro.md (hunk 2 above)
Comment (lempiy) [reply tasks/git-intro.md]: @suchov its magically gone :)

File: contents.md (hunk 3 above)
Comment (lempiy) [reply contents.md]: @suchov its magically gone :)

**Pull request merged**